src/Component/Navbar/styles.scss

```
@@ -0,0 +1,73 @@
+@import "../../resources/variables.scss";
+
+.navbar {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 46px;
+  background-color: $NAVBAR_BG_COLOR;
+  gap: 20px;
+  box-sizing: border-box;
+  justify-content: space-between;
+  padding: 0 10px;
+}
+
+.left_container {
+  display: flex;
+
+  .symbol {
+    margin: auto;
+    cursor: pointer;
+  }
+  .hambur_symbol {
+    display: flex;
+    cursor: pointer;
+  }
+}
+
+.navbar .title {
+  font-size: $NAVBAR_TEXT_SIZE;
+  color: $BLACK;
+  letter-spacing: $TITLE_LETTER_SPACE;
+  font-weight: 200;
+  display: none;
+}
+
+.navbar .symbol_container {
+  display: flex;
+  justify-content: flex-end;
+  gap: 20px;
+
+  .symbol {
+    cursor: pointer;
+  }
+}
+
+.button_container {
+  display: none;
+  gap: 10px;
+}
+
+@media (min-width: 564px) {
```

```suggestion
@media (min-width: breakpoints.mobile) {
```

lets also make a mapping object containing all of the breakpoints needed.
apply this change on every media query being used.